src/libs/antares/study/parameters.cpp

```diff
@@ -926,7 +926,11 @@ bool Parameters::loadFromINI(const IniFile& ini, uint version, const StudyLoadOp
     // A temporary buffer, used for the values in lowercase
     String value;
     String sectionName;
-    typedef bool (*Callback)(Parameters&, const String&, const String&, const String&, uint);
+    typedef bool (*Callback)(Parameters&, // [out] Parameter object to load the data into
+            const String&, // [in] Key, comes left to the '=' sign in the .ini file
+            const String&, // [in] Lowercase value, comes right to the '=' sign in the .ini file
+            const String&, // [in] Raw value as writtent right to the '=' sign in the .ini file
+            uint); // [in] Version of the simulator (such as 710)
```

Comment:
Upon closer examination, the version is read from the **study**, it is not the version of Antares Simulator.

**load.cpp**
```cpp
bool Study::internalLoadHeader(const String& path)
{
    // Header
    buffer.clear() << path << SEP << "study.antares";
    if (not header.loadFromFile(buffer))
    {
        logs.error() << path << ": impossible to open the header file";
        return false;
    }
// ...
}
```
**header.cpp**
```cpp
bool StudyHeader::internalLoadFromINIFile(const IniFile& ini, bool warnings)
{
    version = 0;
    const IniFile::Section* sect = ini.find("antares");
// ...
            // Version
            if (p->key == "version")
            {
                version = p->value.to<uint>(); // Property "version" in section "antares" from file study.antares
                continue;
            }
// ...
}
```